### DISCUSSION TITLE
请教一个问题，Matrix里面的setRotate怎么调用？

### DISCUSSION THREAD

**Discussion by leohandsome1688** [Tencent/libpag#1998] [Q&A] [answered]
文档看了半天没用看明白，也没用找到想对应的demo，我就是像给某个图层旋转，移动啥的，但是不管怎么写，就是说 setRotate 不是一个方法？或者报 ：wasmIns is undefined 

哪位大哥要是知道怎么使用，指点一下，谢谢了

**Comment by leohandsome1688** [comment]
真心的希望demo能够完善？或者说文档能够清晰一点。

**Comment by kevingpqi123** [comment, marked as answer]
图形 Matrix 变化是业界通用的，不是 pag 单独提供的能力，在网上搜索下使用范例一大堆：https://m.xp.cn/b.php/81024.html
或者看下 iOS demo 中的这个范例：https://github.com/libpag/pag-ios/blob/master/sample/pag-ios/ViewController/MoveAndRotateLayerViewController.swift
